hypershift-operator/controllers/nodepool/kubevirt/kubevirt.go

```diff
@@ -147,65 +159,50 @@ func virtualMachineTemplateBase(image string, kvPlatform *hyperv1.KubevirtNodePo
 		},
 	}
 
+	if kvPlatform.RootVolume != nil {
+		if kvPlatform.RootVolume.Persistent != nil && kvPlatform.RootVolume.Persistent.Size != nil {
+			volumeSize = *kvPlatform.RootVolume.Persistent.Size
+		}
+
+		if kvPlatform.RootVolume.Persistent != nil {
+			storageClass = kvPlatform.RootVolume.Persistent.StorageClass
```

Comment:
this is a bit odd to read. It looks like the storageclass is always defined, but in reality the storage class is optional, and will often be `nil` here.

Reply:
That was on purpose. I Made is more readable now.